analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/LogAnalyzerTest.cs

```diff
@@ -99,6 +105,15 @@ private static void VerifyRoslynVersion(IEnumerable<LogInfo> messages)
             version.Should().BeGreaterThan(new Version(3, 0));  // Avoid 1.0.0.0
         }
 
+        private static void VerifyLanguageVersion(IEnumerable<LogInfo> messages)
+        {
+            messages.Should().NotBeEmpty();
+            var versionMessage = messages.SingleOrDefault(x => x.Text.Contains("Language version"));
+            versionMessage.Should().NotBeNull();
+            versionMessage.Severity.Should().Be(LogSeverity.Info);
+            versionMessage.Text.Should().MatchRegex(@"^Language version: (CSharp|VisualBasic)\d");
```

Comment:
```suggestion
            versionMessage.Text.Should().MatchRegex(@"^Language version: (CSharp|VisualBasic)\d+$");
```

Reply:
Enum values can get more tricky, so the general regex should be enough:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.languageversion?view=roslyn-dotnet-3.10.0

https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.visualbasic.languageversion?view=roslyn-dotnet-3.10.0